tests/test_save.py

```diff
@@ -131,3 +144,76 @@ def test_model_manipulation(request, model_class, storage_method):
     finally:
         if os.path.exists(model_fname):
             os.remove(model_fname)
+
+class CustomMlpPolicy(FeedForwardPolicy):
+    """A dummy "custom" policy to test out custom_objects"""
+    def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, reuse=False, **_kwargs):
+        super(CustomMlpPolicy, self).__init__(sess, ob_space, ac_space, n_env, n_steps,
+                                              n_batch, reuse, feature_extraction="mlp",
+                                              **_kwargs)
+
+
+@pytest.mark.parametrize("model_class", MODEL_LIST)
+def test_save_custom_objects(request, model_class):
+    """
+    Test feeding custom_objects in model.load(...) function
+    """
+    # Skip DQN (not an actor-critic policy)
+    if model_class == DQN:
```

Comment:
Looking at the code, I was wondering why there is no continuous model (like DDPG, SAC or TD3) in the previous test. After some checks, this is done in `test_continuous.py`.

Because the `custom_objects` implementation is in the base class, I think testing on common actor critic models should be sufficient. Any thought @hill-a?

Reply:
`test_save.py` uses only discrete environment so SAC/TD3 can not be tested there just by adding it them in the list. Fixing this would be trivial (small if-else check), but if it is already done in `test_continuous.py` then I think it is all good?